pango/src/functions.rs

```diff
@@ -10,18 +8,9 @@ use crate::{Analysis, GlyphString, Item};
 
 #[doc(alias = "pango_reorder_items")]
 pub fn reorder_items(logical_items: &[&Item]) -> Vec<Item> {
```

Comment:
```suggestion
pub fn reorder_items(logical_items: impl IntoIterator<Item = Item>) -> glib::List<Item> {
```

maybe? Or even

```suggestion
pub fn reorder_items(logical_items: glib::List<Item>) -> glib::List<Item> {
```

Reply:
I am ok with either... as I mentioned I never used this API, so I am not sure how the caller gets the list, so I am not sure if taking a glib::List will be annoying

Reply:
Me neither but that's what you'd have in C so it's probably not wrong